### ISSUE TITLE
Context menu sub list

### ISSUE THREAD

**Issue by yairtawil** [AnSyn/ansyn#707]
**I'm submitting a ...**  (check one with "x")

```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
[ ] Task/UserStory
```

**Current behavior** 

Context menu sub list should filtered by sensorName

**Expected behavior** 

Context menu sub list should filtered by sensorType

**Comment by yairtawil** [comment]
resolved by #709

**Comment by yairtawil** [comment]
resolved by #709
